pkg/api/v1/resource_helpers.go

```diff
@@ -62,22 +62,13 @@ func (self *ResourceList) NvidiaGPU() *resource.Quantity {
 	return &resource.Quantity{}
 }
 
-func GetContainerStatus(statuses []ContainerStatus, name string) (ContainerStatus, bool) {
+func GetContainerStatusInfo(statuses []ContainerStatus) map[string]ContainerStatus {
```

Comment:
you are 99% right,but i think we maybe use this function at the same time use old function.